src/main/java/tests/UiTest.java

```diff
@@ -2,15 +2,32 @@
 
 import domain.pages.DirectoryPage;
 
-import java.sql.DriverManager;
+import java.lang.System;
+
+import domain.pages.HomePage;
+import org.junit.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.*;
+import org.openqa.selenium.support.PageFactory;
+
 
 public class UiTest {
 
-    WebDriver driver;
 
-    DirectoryPage directoryPage = PageFactory.initElements(driver, DirectoryPage.class);
+    @Test
+    public void listByStateandDistrict() {
```

Comment:
rename method include should into the method name

Reply:
done